### PR TITLE
Accept requests to '/' and send a plain text response.

### PR DESCRIPTION
**Pull request by Nifdee**
Accepts a request to the root path. Useful for a quick check that the server is up.

### CODE REVIEW

File: typedjsonrpc/server.py [hunk 1]
```diff
@@ -73,6 +77,9 @@ def _dispatch_request(self, request):
         endpoint, _ = adapter.match()
         if endpoint == self._endpoint:
             return self._dispatch_jsonrpc_request(request)
+        elif endpoint == ROOT_ENDPOINT_NAME:
+            return Response(request.method.__repr__() + " is alive!",
```
Comment (mnazbro) [inline typedjsonrpc/server.py]: `repr(request.method)` does the same thing and is the suggested way of doing it.

File: tests/test_server.py [hunk 2]
```diff
@@ -119,6 +119,20 @@ def test_wsgi_app_dispatch(self):
         server(environ, mock_start_response)
         mock_registry.dispatch.assert_called_once_with(mock.ANY)
 
+    def test_wsgi_request_to_root_endpoint(self):
```
Comment (mnazbro) [inline tests/test_server.py]: This test should also check that the response you get is what is expected. Right now it's basically testing that we do not raise an exception when we pass in a message to the root path.

**Comment by mnazbro** [comment]
Please add documentation in the readme.

**Comment by punya** [comment]
This doesn't seem right - it's a requirement for Abacus, not for typedjsonrpc, that we respond to to GET /.

**Comment by mnazbro** [comment]
That's a good point. We should probably allow you to just add arbitrary endpoints in `Server`, and then we can implement the root plain return elsewhere.

**Pull request closed without merging**